openapi/generator.py

```diff
@@ -265,6 +268,7 @@ def generate_attr_update_classes(self, class_name, identation):
             "Primitive": [("set", "Any")],
             "Object": [("set", "Dict"), ("add", "Dict"), ("remove", "List")],
             "List": [("set", "List"), ("add", "List"), ("remove", "List")],
+            "Label": [("add", "List"), ("remove", "List")],
```

Comment:
@fredrik955 this might be a bit too general. For example, will we ever want Labels on `ThreeDModelRevision`

Reply:
Even though we add the class there doesn't mean it will be exposed/used. We add the ObjectPatchClass in many resources without them using it. The generator doesn't care if it is being used as the property definition is defined another place in the generator.

Reply:
Fair enough! 👍